modules/strelka/1.0/config/default.yaml

```diff
@@ -0,0 +1,39 @@
+lcr-modules:
+    
+    strelka:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            sample_bam: null 
+            sample_bai: null
+            candidate_indel: null
+
+        scratch_subdirectories: []
+
+        options:
+            configure:
+                mrna: "--rna"
+                capture: "--exome"
+            strelka: "-m local"
+
+        conda_envs:
+            strelka: "{MODSDIR}/envs/strelka-2.9.10.yaml"
+            
+        threads:
+            strelka: 4
+
+        mem_mb:
+            strelka: 2000
+
+        pairing_config:
+            genome:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            capture:
+                run_paired_tumours: True
+                run_unpaired_tumours_with: "unmatched_normal"
+                run_paired_tumours_as_unpaired: False
+            mrna:
+                run_paired_tumours: False
+                run_unpaired_tumours_with: "no_normal"
+                run_paired_tumours_as_unpaired: True
```

Comment:
I don't think Strelka is typically run on RNA-seq data, so I think you can delete this section.

Reply:
I noticed that you have rules to run Strelka in tumour-only mode or rather germline mode. Is this something that the CLC does?

Reply:
The snakefile I got from Lauren runs strelka on tumour-normal pairs only. I found the germline workflow but wasn't sure if it is tumour-only. Have you used it before?